docs/tutorials/plots.rst

```diff
@@ -51,15 +49,23 @@ These take, at minimum, `astroplan.Observer`, `astroplan.FixedTarget` and
 
 .. _plots_airmass:
 
-Airmass versus time plots are made in the following manner::
+Airmass vs time plots are made the following way:
+
+.. code-block:: python
 
-    plot_airmass(target, observer, time)
+    >>> from astroplan.plots import plot_airmass
+
+    >>> plot_airmass(target, observer, time)
```

Comment:
Looks like this line is causing the Travis CI build to fail--the doctests are looking for something named "target", which has not yet been constructed.  Only way I know to take care of this is to skip doc tests for individual code blocks until no more errors.